Tests/FluentAssertions.Specs/Xml/XDocumentAssertionSpecs.cs

```diff
@@ -844,9 +844,11 @@ public void When_asserting_document_has_root_element_and_it_does_it_should_succe
         {
             // Arrange
             var document = XDocument.Parse(
-                @"<parent>
-                    <child />
-                  </parent>");
+                """
+                    <parent>
+                        <child />
+                    </parent>
+                """);
```

Comment:
How about we back indent this a level?

```suggestion
                """
                <parent>
                    <child />
                </parent>
                """);
```